spec/requests/contact_phone_forms_spec.rb

```diff
@@ -56,12 +56,14 @@
         context "when valid params are submitted" do
           let(:valid_params) {
             {
-              reg_identifier: transient_registration[:reg_identifier]
+              reg_identifier: transient_registration[:reg_identifier],
+              phone_number: "01234 567890"
```

Comment:
Does this work (using 01234 567890)?

When I try this on the WEX service I get "This telephone number is not recognised"

Reply:
It validates for me – that's what we've been using for the seed data as well.

Reply:
🤦‍♂️ 
<img width="508" alt="screen shot 2018-04-10 at 09 35 40" src="https://user-images.githubusercontent.com/1789650/38545728-9dec28d4-3ca2-11e8-9d1e-b8b7061b7b0e.png">

...or "why it may be wise to not review PR's late at night!"

Reply:
Hahaha, we've all been there...